src/presentation/http/router/oauth.ts

```diff
@@ -34,7 +34,48 @@ const OauthRouter: FastifyPluginCallback<OauthRouterOptions> = (fastify, opts, d
   /**
    * Callback for Google oauth2. Google redirects to this endpoint after user authentication.
    */
-  fastify.get('/google/callback', async (request, reply) => {
+  fastify.get('/google/callback', {
+    schema: {
+      params: {
+        clientId: {
+          $ref: 'OauthSchema#/properties/clientId',
+        },
+        clientSecret: {
+          $ref: 'OauthSchema#/properties/clientSecret',
+        },
+      },
+
+      response:{
+        '2xx': {
```

Comment:
I think you can move it to separate file like we do it here https://github.com/codex-team/notes.api/tree/main/src/presentation/http/schema

Reply:
there's already a separate file

Reply:
I don't see a separate file for the schema of oauth